src/UserLabel/UserLabel.stories.tsx

```diff
@@ -47,7 +48,17 @@ export const CustomColor = {
 // Long Name
 export const LongName = {
   args: {
-    label: "James withAVeryLooooooooooongMiddleName Harper"
+    label: "James withAVeryLooooooooooongMiddleName Harper",
+    tooltip: "Created by"
+  },
+  render: Template
+};
+
+// With Tooltip
+export const WithTooltip = {
+  args: {
+    label: "John Doe",
+    tooltip: "Created by"
```

Comment:
I think our convention is to use all props in the default story, thus here we should show no tooltip in my opinion.

```suggestion
// Without Tooltip
export const WithoutTooltip = {
  args: {
    label: "John Doe"
```